go-controller/pkg/ovn/egressfirewall_test.go

```diff
@@ -611,6 +618,238 @@ var _ = ginkgo.Describe("OVN EgressFirewall Operations", func() {
 				gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 			})
+			ginkgo.It(fmt.Sprintf("egress firewall with node selector updates during node update, gateway mode %s", gwMode), func() {
+				config.Gateway.Mode = gwMode
```

Comment:
you don't need that, it is set in the beginning of the loop

Reply:
actually this whole for loop doesn't work properly with setting up gwMode, but I can update it in another pr

Reply:
https://github.com/ovn-org/ovn-kubernetes/pull/3449/commits/2292961eab87455ffafa05045ad79c5061afa167